src/python_testing/matter_testing_infrastructure/chip/testing/spec_parsing.py

```diff
@@ -512,56 +513,74 @@ class PrebuiltDataModelDirectory(Enum):
     k1_4 = auto()
     kMaster = auto()
 
+    @property
```

Comment:
You can eliminate this if you do
```
PrebuildDataModelDirectory(str, Enum):
  k1_3 = "1.3"
  k1_4 = "1.4"
  kMaster = "master"
```
^ then just print the enum value directly.

Reply:
ditto below

Reply:
The pre-existing code uses that - was that changed to solve a particular problem?

Reply:
Previous code was freely assuming that `str` can be used and I thought that was not ideal. I changed it to typed enums on purpose and that also allows us to do `isinstance` checks on things.

Reply:
I think forcing explicit types here is better as there is some confusion on strings if they are `paths` or `built in versions`. We could make the code have heuristics to pick what they think is best, however that seems more brittle. I would prefer to enforce strict typing of Traversable (i.e. `pathlib.Path(path)`) or Prebuilt constants (that are not strings).